icechunk/Cargo.toml

```diff
@@ -57,3 +63,6 @@ tempfile = "3.15.0"
 
 [lints]
 workspace = true
+
+[features]
```

Comment:
we have an optional feature now, that creates a function to enable tracing. Apps using Icechunk as a rust library don't want this, they want to do their own tracing initialization.